libs/features/mas/src/merch-card.js

```diff
@@ -43,6 +43,7 @@ export class MerchCard extends LitElement {
         actionMenu: { type: Boolean, attribute: 'action-menu' },
         customHr: { type: Boolean, attribute: 'custom-hr' },
         consonant: { type: Boolean, attribute: 'consonant' },
+        spectrum: { type: String, attribute: 'spectrum' }, /* css|swc */
```

Comment:
maybe we want to reflect both spectrum and consonant attributes?

Reply:
I think, both `consonant` and `spectrum` attributes should be there only when they are set externally.
I want to keep the markup simple for the default use case.

Reply:
okie agreed